plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -560,10 +560,9 @@ def _sendNextGcodeLine(self):
             line = line[:line.find(";")]
         line = line.strip()
 
-        # Don't send empty lines
+        # Don't send empty lines. But we do have to send something, so send m105 instead.
         if line == "":
-            self._gcode_position += 1
-            return
+            line = "M105"
```

Comment:
Ah, I would have run into this issue soon then, because I have just received a smoothieboard clone for a new "project".